docs/catalog/web/serve-images-in-modern-formats.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: dubrie
+published_date: TBD
+category: web
+tags: 
+ - image
+ - size:small
+---
+
+# Serve images in Moden Formats
+
+## Description
+
+Images are one of the main parts of modern Websites and used almost on every website. Modern Image formats can help to reduce bandwith, storage and computing requirements on the displaying device.
+
+There are serveral new image formats to choose from, for example WebP and AVIF. All Formats provide better compression ratio, increased quality and more features as traditional formats.
+
+
+## Solution
+
+Validate which image format is supports the use case of the web application the best. Convert images to modern image formats and deliver them to the displaying devices. 
```

Comment:
Is there any guidance on how to validate image formats?

Reply:
Yes. At least: validate if image size actually decreases (one could have the worst case of images in the project), if the target browser supports the new format and if the image quality is as expected.
On the other side we are not targeting specific applications. So I thought that a project which, for example, implements the next google earth would have many more requirements (which I can not think of). 

Do you think we should state a few validation points as guideline? Most of the patterns I saw where really short, so I converted them to short (1, 2 or 3 sentences) patterns as well.